src/WL/ServerProtocol.hsc

```diff
@@ -0,0 +1,46 @@
+{-# LANGUAGE PatternSynonyms #-}
+module WL.ServerProtocol where
+
+import Foreign.C.Types (CInt)
+import Foreign.C.Types
+
+#include <wayland-server-protocol.h>
+
+
+type WL_data_device_manager_dnd_action = CInt
```

Comment:
Can this not be rewritten using `{{ enum … }}`?

Reply:
ah okay it uses the value defined in the C code

I think at the time `enum` could only be used for sequences like 1, 2, 3, 4...

and this one is doing the C flag thing where they are all powers of two.

Reply:
Yep… at one time I thought the same, but @dylan-hachmann corrected me on that point.